### PR TITLE
Fix translatable messages

### PR DESCRIPTION
**Pull request by shawnadelic**
Fix minor message construction and mistranslation issues with translatable message strings where found in various apps.

Refs SHOOP-1985

### CODE REVIEW

**Comment by akx** [comment]
Could you move the commit descriptions to the title lines?

Just

> admin: Fix translatable messages

and

> addons: Fix translatable messages

etc. would be fine! :)

File: shoop/addons/templates/shoop/admin/addons/reload.jinja [hunk 1]
```diff
@@ -41,7 +41,7 @@
     function beginCheckReload() {
         stopCheckReload();
         reloadCheckTimer = setInterval(checkReload, 2000);
-        Messages.enqueue({tags: "info", text: "Reload initiated."});
+        Messages.enqueue({tags: "info", text: gettext("Product")});
```
Comment (akx) [inline shoop/addons/templates/shoop/admin/addons/reload.jinja]: Copy-paste error? :)

File: shoop/admin/templates/shoop/admin/system/telemetry.jinja [hunk 2]
```diff
@@ -25,7 +25,7 @@
                     {% if is_grace and opt_in %}
                         <p><i class="text-info fa fa-info-circle"></i> {% trans %}Telemetry submission is currently suspended; data will only be sent 24 hours after installation if still opted in at that point.{% endtrans %}</p>
                     {% elif last_submission_time %}
-                        <p>{% trans %}Telemetry data was last submitted at{% endtrans %} {{ last_submission_time|datetime }} - ({{ last_submission_time|timesince }} {% trans %}ago{% endtrans %})</p>
+                        <p>{{ _("Telemetry data was last submitted at %s - (%s) ago" % (last_submission_time|datetime, last_submission_time|timesince)) }}</p>
```
Comment (akx) [inline shoop/admin/templates/shoop/admin/system/telemetry.jinja]: This should probably just use a regular `{% trans %}` block?

Also, if you use the `_` function, you shouldn't use % interpolation; instead use named placeholders and kwargs, like

```jinja2
{{ _('Hello %(name)s!', name='World') }}
```

File: shoop/themes/classic_gray/templates/classic_gray/shoop/user/login.jinja [hunk 3]
```diff
@@ -11,7 +11,7 @@
             {% trans %}Log in{% endtrans %}
         </h1>
         <p class="lead">
-            {% trans %}You have already been logged in as{% endtrans %} <strong>{% if user.first_name or user.last_name %}{{ user.get_full_name() }}{% else %}{{ user }}{% endif %}</strong>
+            {% trans username = user.get_full_name() if (user.first_name or user.last_name) else user %}You have already been logged in as <strong>{{ username }}</strong>{% endtrans %}
```
Comment (akx) [inline shoop/themes/classic_gray/templates/classic_gray/shoop/user/login.jinja]: I guess `You have already logged in as ...` would be nicer phrasing?

File: shoop/addons/templates/shoop/admin/addons/reload.jinja (hunk 1 above)
Comment (shawnadelic) [inline shoop/addons/templates/shoop/admin/addons/reload.jinja]: -_-, thanks!

**Comment by tulimaki** [comment]
retest this please

**Comment by tulimaki** [comment]
test this please

**Comment by tulimaki** [comment]
ok to test

**Comment by tulimaki** [comment]
test this please

**Comment by suutari-ai** [comment]
retest this please

**Pull request merged**